frontend/components/MobileAccordion.vue

```diff
@@ -0,0 +1,84 @@
+<template>
+  <div class="p-4">
+    <!-- Mobile View -->
+    <div class="md:hidden">
+      <div
+        v-for="item in menuItems"
+        :key="item.title"
+        class="border-b-2 border-gray-200 py-2"
+      >
+        <button
+          class="w-full flex justify-between items-center"
+          @click="toggleItem(item.title)"
+        >
+          <span class="font-bold italic text-offblack">{{ item.title }}</span>
+          <span class="text-xl px-2">
+            {{ isActive(item.title) ? '-' : '+' }}
+          </span>
+        </button>
+        <transition name="fade">
+          <div v-if="isActive(item.title)" class="ml-4 mt-2">
+            <div
+              v-for="(section, key) in item.sections"
+              :key="`${section.title}-${key}`"
+            >
+              <div class="font-bold italic text-offblack">
+                {{ section.title }}
+              </div>
+              <ul>
+                <li
+                  v-for="(subItem, subItemKey) in section.subItems"
+                  :key="`${section.title}-${key}-${subItemKey}`"
```

Comment:
It's important to clearly identify the items so the Vue virtual DOM doesn't get confused. I believe we should have included an identifier in the data structure from the start.